readme.md

```diff
@@ -806,6 +806,7 @@
 - [@sindresorhus/is](https://github.com/sindresorhus/is) - Type check values.
 - [env-dot-prop](https://github.com/simonepri/env-dot-prop) - Get, set, or delete nested properties of process.env using a dot path.
 - [emittery](https://github.com/sindresorhus/emittery) - Simple and modern async event emitter.
+- [esm](https://github.com/standard-things/esm) - Tomorrow's ECMAScript modules today!
```

Comment:
This is not a useful description. Describe what it is and does. No marketing tagline.